packages/core/helper-plugin/lib/src/components/FilterPopoverURLQuery/index.js

```diff
@@ -161,12 +163,21 @@ const FilterPopoverURLQuery = ({ displayedFilters, isVisible, onBlur, onToggle,
             </Box>
             {operator !== '$null' && operator !== '$notNull' && (
               <Box>
-                <Inputs
-                  {...appliedFilter.metadatas}
-                  {...appliedFilter.fieldSchema}
-                  value={modifiedData.value}
-                  onChange={(value) => setModifiedData((prev) => ({ ...prev, value }))}
-                />
+                {CustomInput ? (
+                  <CustomInput
```

Comment:
How about something like:

```js
import DefaultInputs from './inputs'

const Inputs = CustomInput || DefaultInputs

return <Inputs {...} />
```

and making sure CustomInput matches the signature of the default inputs (therefore removing setModifiedData from ComboboxFilter)